CMakeLists.txt

```diff
@@ -99,6 +99,27 @@ option(SLANG_ENABLE_SLANG_GLSLANG "Enable glslang dependency and slang-glslang w
 option(SLANG_ENABLE_TESTS "Enable test targets, some tests may require SLANG_ENABLE_GFX, SLANG_ENABLE_SLANGD or SLANG_ENABLE_SLANGRT" ON)
 option(SLANG_ENABLE_EXAMPLES "Enable example targets, requires SLANG_ENABLE_GFX" ON)
 
+option(EXTERNAL_MINIZ "Build using external Miniz library" OFF)
+option(EXTERNAL_LZ4 "Build using external LZ4 library" OFF)
+option(EXTERNAL_VULKAN_HEADERS "Build using external Vulkan headers" OFF)
+option(EXTERNAL_SPIRV_HEADERS "Build using external SPIR-V headers" OFF)
+option(EXTERNAL_UNORDERED_DENSE "Build using external unordered dense" OFF)
```

Comment:
Please can you prefix any user facing variables with `SLANG_`

Reply:
For consistency with the llvm options, perhaps call these like `SLANG_USE_SYSTEM_MINIZ` etc..?